package.json

```diff
@@ -35,7 +35,7 @@
     "build:template": "handlebars \"src/formatter/templates/$TEMPLATE.handlebars\" -f \"src/formatter/templates/$TEMPLATE.js\" --known each --known if --known with --known paragraphClasses --known isChordLyricsPair --known isTag --known isComment --known shouldRenderLine --known hasChordContents --known lineHasContents --known lineClasses --known toUpperCase --known paragraphClasses --commonjs handlebars",
     "build:templates": "TEMPLATE=html_div_formatter yarn build:template && TEMPLATE=html_table_formatter yarn build:template",
     "build:sources": "rm -rf lib && babel src --out-dir lib",
-    "build:pegjs": "pegjs -o src/parser/chord_pro_peg_parser.js src/parser/chord_pro_grammar.pegjs",
+    "build:pegjs": "peggy -o src/parser/chord_pro_peg_parser.js src/parser/chord_pro_grammar.pegjs && peggy src/parser/on_song_grammar.pegjs",
```

Comment:
@martijnversluis it looks like pegjs has been abandoned, and some of the contributors have revived a fork of it as [peggy](https://github.com/peggyjs/peggy) with some bug fixes and improvements.  One really nice feature that peggy has added is the `@` operator to "pluck" a value out of a group. For example `("{" @TagName "}")` will just return the value of `@TagName` instead of requiring a label and function to pluck it out of the match.

The existing grammar builds and passes tests without changes using peggy. Are you ok with switching to this, and do you want me to open a separate PR with just this change?